README.md

```diff
@@ -66,6 +66,8 @@ Below are links to profiles where you can see Readme Typing SVGs in action!
 [![Raihan Khan](https://github.com/raihankhan.png?size=60)](https://github.com/raihankhan)
 [![notauserx](https://github.com/notauserx.png?size=60)](https://github.com/notauserx)
 [![wiz64](https://github.com/wiz64.png?size=60)](https://github.com/wiz64)
+[![Aquarian Blake](https://github.com/Aquarius-blake.png?size=60)](https://github.com/Aquarius-blake)
+
```

Comment:
Seems like an extra empty line was added, besides that, it looks good
```suggestion
[![Aquarian Blake](https://github.com/Aquarius-blake.png?size=60)](https://github.com/Aquarius-blake)
```

Reply:
Sorry about the empty line I'll try to rectify that, thanks though